addon/components/sl-calendar.js

```diff
@@ -14,40 +13,31 @@ export default Ember.Component.extend({
     // -------------------------------------------------------------------------
     // Attributes
 
-    /**
-     * Class names for the root element
-     *
-     * @property {Ember.Array} classNames
-     */
+    /** @type {String[]} */
+    classNameBindings: [ 'locked:sl-calendar-locked' ],
+
+    /** @type {String[]} */
     classNames: [ 'sl-calendar' ],
 
-    /**
-     * Bindings for the component's class names
-     *
-     * @property {Ember.Array} classNameBindings
-     */
-    classNameBindings: [ 'locked:sl-calendar-locked' ],
+    /** @type {Object} */
+    layout,
 
     // -------------------------------------------------------------------------
     // Actions
 
-    /**
-     * Object of actions
-     *
-     * @property {Ember.Object} actions
-     */
+    /** @type {Object} */
     actions: {
 
         /**
          * Change the currently-viewed decade by incrementing or decrementing
          * the decadeStart year number
          *
-         * @function actions.changeDecade
-         * @param    {number} decadeMod - A number to adjust the decadeStart by
-         *           (positive to increment, negative to decrement)
-         * @returns  {void}
+         * @function actions:changeDecade
+         * @param {Number} decadeMod - A number to adjust the decadeStart by
```

Comment:
Do we want a "-" separating the param name and the description?  Think this will cause the description to then display as "- ..." in the generated documentation.